agent/containers/images/Dockerfile.layered.j2

```diff
@@ -10,5 +10,14 @@ COPY ./{{ distro }}-pcp.repo /etc/yum.repos.d/pcp.repo
 # FIXME: this is not exhaustive, it does not include RPMs to support
 #        Kubernetes or RHV environments.
 RUN {% if distro == 'centos-7' %}yum{% else %}dnf{% endif %} install -y --setopt=tsflags=nodocs {% if distro == 'centos-8' %}--enablerepo powertools {% endif %}{% if kind in ('tools', 'all') %}--enablerepo pcp {% endif %}{{ rpms }} && \
+    {% if distro == 'centos-7' %}yum{% else %}dnf{% endif %} -y clean all && \
```

Comment:
I'd check with Ashish, but I don't think we can include DCGM in our containers we publish.

We can provide a separate docker file that shows how to add DCGM as an example, but it can't be part of the base images.